daemon/oci_linux.go

```diff
@@ -911,6 +926,31 @@ func (daemon *Daemon) createSpec(c *container.Container) (retSpec *specs.Spec, e
 		s.Linux.ReadonlyPaths = c.HostConfig.ReadonlyPaths
 	}
 
+	// We just need to passthrough two types of annotations to runtime
+	// first is user specific annotations which with prefix of `annotations.`
+	// the other is labels for docker used which is marked as internal.
+	for k, v := range c.Config.Labels {
+		logrus.Info(k)
+		if oldValue, ok := s.Annotations[k]; ok {
+			return nil, fmt.Errorf("Key %q already exisit in Annotations, the new value will be ignored,"+
```

Comment:
This function have only one return value, why we return 2 values here, am I missing sth?

Reply:
func returns `spec` and error

Reply:
https://github.com/moby/moby/blob/master/daemon/oci_linux.go#L37 , this one❓

Reply:
@arm64b I think we are talking about https://github.com/moby/moby/blob/c752b0991e31ba9869ab6a0661af57e9423874fb/daemon/oci_linux.go#L747

Reply:
Ah, yes, you are right 😄